compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/SymbolFactory.java

```diff
@@ -213,7 +213,7 @@ public static ParameterSymbol createBallerinaParameter(BVarSymbol symbol, Parame
         if (symbol == null) {
             return null;
         }
-        String name = symbol.getName().getValue();
+        String name = symbol.getName().getValue().isBlank() ? null : symbol.getName().getValue();
```

Comment:
`.isBlank()` will capture cases such as "      "(empty spaces) as well. Do we really need this or just would `isEmpty()` is suffcient?

Reply:
Yes, name should never be empty, and since the `isBlank` internally checks for both isEmpty and space chars, therefore for the safe hand added this.

Reply:
My argument is will you ever get a symbol name with empty spaces? `isBlank` internally traverses through the whole char-array where `isEmpty` only checks char array `length == 0`.